File: Estrutura_Sequencial/JavaScript/Ex1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};

var nome = prompt();
var salario = parseFloat(prompt());
var vendas = parseFloat(prompt());

total = salario + (vendas * 0.15);
console.log("TOTAL = R$", total.toFixed(2));

File: Estrutura_Sequencial/C#/Ex1009.cs
using System;

class MainClass {
    public static void Main (string[] args) {

        string nome = Console.ReadLine();
        float salario = float.Parse(Console.ReadLine());
        float vendas = float.Parse(Console.ReadLine());

        float salariof = salario + (float)(vendas * 0.15);
        Console.WriteLine($"TOTAL = R$ {salariof:F2}");
    }
}